src/Trakx.CoinGecko.ApiClient/CoinGeckoClients/CoinGeckoMarketClient.cs

```diff
@@ -50,8 +52,10 @@ public async Task<IDictionary<DateTimeOffset, MarketData>> GetMarketDataForDateR
         CancellationToken cancellationToken = default)
     {
         var date = asOf.ToDateString();
-        var cacheKey = $"{_typeName}|market-data|{id}|{quoteCurrencyId}|{date}";
-        return await GetFromCacheOrApi(cacheKey, async () => await GetMarketDataAsOfFromIdInternal(id, asOf, quoteCurrencyId, date));
```

Comment:
Personally I like it was before :) more readable imo

Reply:
You're right, I have reverted to build the cache key in the caller.